.claude/commands/generate-test.md

```diff
@@ -0,0 +1,200 @@
+# Generate Manual Testing Documentation
+
+Generate an HTML manual testing document from a Ginkgo E2E test file.
```

Comment:
I fail to understand why this is necessary. What is this HTML testing document used for?